data/2015/other/Academic Policies & Procedures.xhtml

```diff
@@ -113,69 +113,70 @@
-<p class="abbreviations-list">F Distance Education-“Field” Class</p>
-<p class="abbreviations-list">fld field</p>
-<p class="abbreviations-list">fund or fun fundamentals</p>
-<p class="abbreviations-list">G graduate</p>
-<p class="abbreviations-list">GPA Grade Point Average</p>
-<p class="abbreviations-list">grad or Grad graduate</p>
-<p class="abbreviations-list">H Honors</p>
-<p class="abbreviations-list">hr or hrs hours</p>
-<p class="abbreviations-list">I or inc incomplete</p>
-<p class="abbreviations-list">ind independent</p>
-<p class="abbreviations-list">intermed intermediate</p>
-<p class="abbreviations-list">intro or intr introduction</p>
-<p class="abbreviations-list">L laboratory with credit hours</p>
-<p class="abbreviations-list">Lab or lab laboratory</p>
-<p class="abbreviations-list">Lec lecture</p>
-<p class="abbreviations-list">max maximum</p>
-<p class="abbreviations-list">NP or N No Pass credit allowed toward degree</p>
-<p class="abbreviations-list">NR No Report</p>
-<p class="abbreviations-list">P Pass</p>
-<p class="abbreviations-list">P/N Pass/No Pass</p>
-<p class="abbreviations-list">PO Pass/No Pass grading option</p>
-<p class="abbreviations-list">parallel denotes taking another class along with the one listed</p>
-<p class="abbreviations-note"><span class="requirement-ital"> (a corequisite or coregistration)</span></p>
-<p class="abbreviations-list">Prereq or preq prerequisite</p>
-<p class="abbreviations-list">princ principles</p>
-<p class="abbreviations-list">Pro or prof professional</p>
-<p class="abbreviations-list">PSI Personal System of Instruction</p>
-<p class="abbreviations-list">quz quiz</p>
-<p class="abbreviations-list">Rec Recreation</p>
-<p class="abbreviations-list">rct or Reci recitation</p>
-<p class="abbreviations-list">rsh research</p>
-<p class="abbreviations-list">S (denotes) Distance Education class</p>
-<p class="abbreviations-list">sec secondary</p>
-<p class="abbreviations-list">sem semester</p>
-<p class="abbreviations-list">smnr seminar</p>
-<p class="abbreviations-list">Stu or stu studio</p>
-<p class="abbreviations-list">TBA to be assigned</p>
-<p class="abbreviations-list">tch teach</p>
-<p class="abbreviations-list">tchr teacher</p>
-<p class="abbreviations-list">tchng teaching</p>
-<p class="abbreviations-list">tech technology or technical</p>
-<p class="abbreviations-list">UG undergraduate</p>
-<p class="abbreviations-list">UNL University of Nebraska–Lincoln</p>
-<p class="abbreviations-list">UNMC University of Nebraska Medical Center</p>
-<p class="abbreviations-list">UNO University of Nebraska at Omaha</p>
-<p class="abbreviations-list">W withdrew</p>
-<p class="abbreviations-list">/ and/or</p>
-<p class="abbreviations-list">&amp; and</p>
-<p class="abbreviations-list">+ and</p>
-<p class="abbreviations-list">&gt; greater than</p>
-<p class="abbreviations-list">&lt; less than</p>
+<p class="requirement-sec-2">ACE. . . . . . .Achievement-Centered Education</p>
```

Comment:
I have no idea why this section was changed. This will not work when it is rendered by the bulletin application.